### DISCUSSION TITLE
Syncing problem after update to 2.1.1

### DISCUSSION THREAD

**Discussion by algo253** [Chia-Network/chia-blockchain#16627] [Support]
Dear Support team, 
After update chia to 2.1.1 i have a syncing problem. Sinching progress stucked close to the final figure. Please see attached screens and let me know how to fix this. 
![chia_1](https://github.com/Chia-Network/chia-blockchain/assets/83923016/c3c1a592-7edf-40df-8fbd-ef0413202dfa)
![Screenshot_2](https://github.com/Chia-Network/chia-blockchain/assets/83923016/d82cc6f5-977e-4cd1-8b8b-b18b9e71c1ca)

**Comment by BrandtH22** [comment]
Hey @algo253 , it appears you have also begun asking this in our discord support channel.
Thank you as that is the place we can generally provide more thorough and timely support (I am ClydeWallace on discord) so we can continue there.

In case you miss the discord message my initial questions are:
Do you have any peers connected?
Do you see any relevant errors or warnings in the log file?

**Comment by BrandtH22** [comment]
This has been resolved in discord:
https://discord.com/channels/1034523881404370984/1034870571864948777/1164235962704998480
